lib/messagebird/client.rb

```diff
@@ -232,6 +232,13 @@ def message(id)
       Message.new(request(:get, "messages/#{id}"))
     end
 
+    # Retrieve messages with optional paging and status filter.
+    def message_list(limit: 10, offset: 0, status: '')
+      query = "messages?limit=#{limit}&offset=#{offset}"
+      query = query + '&status=' + status if status != ''
```

Comment:
Please use https://apidock.com/ruby/URI/encode_www_form/class

Reply:
Good point! Thank you! fixed.